SayLess/__init__.py

```diff
@@ -20,26 +20,35 @@
 import random
 import string
 from datetime import timedelta
+from werkzeug.utils import secure_filename
```

Comment:
Where is werkzeug.utils imported from? I don't see it in requirements and now sure if this is built in.

Reply:
no requirments needed